<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maths5A_CoonsClass</class>
 <widget class="QMainWindow" name="Maths5A_CoonsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math5A ESGI - Coons</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="centralLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupMode">
       <property name="title">
        <string>Mode de génération Béziers</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="2">
         <widget class="QSpinBox" name="spinPrecision">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="rbAdjust">
          <property name="text">
           <string>Réglage de l'hauteur</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="laHorizontal">
          <property name="text">
           <string>Degré horizontal</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ordre de raccordement</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QPushButton" name="bJoin">
          <property name="text">
           <string>Raccorder</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rbRandom">
          <property name="text">
           <string>Aléatoire</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="laVertical">
          <property name="text">
           <string>Degré vertical</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QPushButton" name="bCancelJoin">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="laPrecision">
          <property name="text">
           <string>Nombre de pas (précision)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinHori">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="spinVerti">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QSpinBox" name="spinJoinOrder">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="15" column="1" colspan="2">
         <widget class="QPushButton" name="bResetData">
          <property name="text">
           <string>Réinitialiser les données</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="14" column="1" colspan="2">
         <widget class="QLabel" name="laTimeCalcSurface">
          <property name="text">
           <string>0 us</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupRotation">
       <property name="title">
        <string>Subdivision</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QRadioButton" name="rbKobbelt">
          <property name="text">
           <string>Kobbelt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbCatmull">
          <property name="text">
           <string>Catmull-Clark</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbLoop">
          <property name="text">
           <string>Loop</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="bGenerateCube">
          <property name="text">
           <string>Générer Cube</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="bSubdivide">
          <property name="text">
           <string>Subdiviser</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowWireframe">
       <property name="text">
        <string>Filaire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowTexture">
       <property name="text">
        <string>Texture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowPoints">
       <property name="text">
        <string>Afficher les points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowGrid">
       <property name="text">
        <string>Grille et les axes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupLight">
       <property name="title">
        <string>Lumières</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0">
         <widget class="QCheckBox" name="cbDiffuse">
          <property name="text">
           <string>Diffuse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="bColorS1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="cbSpecular">
          <property name="text">
           <string>Specular</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbLight1">
          <property name="text">
           <string>Source 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="bColorS2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbLight2">
          <property name="text">
           <string>Source 2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="bColorObj">
          <property name="text">
           <string>Couleur du patch</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>bColorS1</zorder>
       <zorder>bColorS2</zorder>
       <zorder>cbLight1</zorder>
       <zorder>cbLight2</zorder>
       <zorder>cbDiffuse</zorder>
       <zorder>cbSpecular</zorder>
       <zorder>bColorObj</zorder>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bResetCam">
       <property name="text">
        <string>Réinitialiser le caméra</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bQuit">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
